mkdocs.yml

```
@@ -39,6 +39,7 @@ nav:
     - REST API Swagger: https://api.dandiarchive.org/swagger
     - REST API Redoc: https://api.dandiarchive.org/redoc
   - DANDI Hub: "50_hub.md"
+  - Real-Time Status: https://www.dandiarchive.org/upptime
```

Great Idea!  what about expanding this also into having 
```suggestion
  - Health Status:
    - Dandisets: https://github.com/dandi/dandisets-healthstatus
    - Services: https://www.dandiarchive.org/upptime
```

---

Thanks @yarikoptic.  I have included your suggestion.